tutorials/algorithms/09_IQPE.ipynb

```diff
@@ -548,20 +583,25 @@
     }
    ],
    "source": [
-    "sim = Aer.get_backend('qasm_simulator')\n",
-    "count0 = sim.run(transpile(qc_S, sim)).result().get_counts()\n",
+    "sampler = Sampler()\n",
+    "\n",
+    "job = sampler.run([qc_S])\n",
```

Comment:
Is it necessary to put the circuit to a list? I know the `Sampler` from qiskit-ibm-runtime handles single circuit as well.

Reply:
No that's not required -- removed!